spec/README.md

```diff
@@ -158,78 +158,78 @@ Example structure for the German language source file, which only contains lower
 
 ###### metadata.alphabet
 
-Required  
```

Comment:
Please note this removal

Reply:
Sorry, what do you mean?

Reply:
You've removed the two blanks to enforce a line break. Or am I missing something?

Reply:
oh my darn editor... I disabled the strip line endings, but it's still doing it.

Reply:
You're using Atom, right? I've started using PHPStorm a few months ago and it's really awesome. All the pain I had with Atom is gone. It just works, without needing to patch everything, starting with language dependent keyboard shortcuts.